components/core/CMakeLists.txt

```diff
@@ -105,37 +105,72 @@ else()
     message(FATAL_ERROR "Could not find ${CLP_LIBS_STRING} libraries for Boost")
 endif()
 
-# Find and setup fmt
-# NOTE:
-# - We only try to link to the static library
-# - spdlog uses fmt, so their versions need to be kept in-sync
-find_package(fmt 8.0.1 REQUIRED)
-if(fmt_FOUND)
-    message(STATUS "Found fmt ${fmt_VERSION}")
-else()
-    message(FATAL_ERROR "Could not find static libraries for fmt")
-endif()
+# Find and setup spdlog and fmt transitive dependency
 
-# Find and setup spdlog
+# Specify the minimum required spdlog version (can be overridden by the user)
```

Comment:
The user shouldn't really be allowed to override the version since they could easily pick a version that's incompatible with the code, right?